src/unix/bsd/apple/mod.rs

```diff
@@ -3358,18 +3358,10 @@ pub const DLT_LOOP: ::c_uint = 108;
 pub const BPF_ALIGNMENT: ::c_int = 4;
 
 // sys/mount.h
-pub const MNT_RDONLY: ::c_int = 0x00000001;
```

Comment:
just curious any reason removing them ?

Reply:
I'm not removing them altogether, just moving them into bsd/mod.rs .  Those constants have the same value on all of the BSDs.

Reply:
ah right